### ISSUE TITLE
training

### ISSUE THREAD

**Issue by hxx160011** [weili1457355863/VPS-Net#13]
Can you provide the training script, thank you!

**Comment by hs7251912** [comment]
VPS-Net is follow this project to train : https://github.com/eriklindernoren/PyTorch-YOLOv3
you should make custom parking slot dataset to train,
annotations format : [classid     x_center     y_center   bbox_w   bbox_h   vacant] 

class: 

right-angled_head  : 0

acute-angled_head  :  1

obtuse-angled_head : 2

marking_point : 3

x_center:

y_center:

bbox_w:

bbox_h:

whether parking slot is vacant:

vacant : 0

non-vacant : 1

marking point : 2 

angle: 67, 90 , 127

**Comment by hxx160011** [comment]
Thank you very much！Can i ask about the loss definition of&nbsp; the Cutomized Alexnet?




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2021年9月18日(星期六) 下午3:50
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [weili1457355863/VPS-Net] training (#13)





 
VPS-Net is follow this project to train : https://github.com/eriklindernoren/PyTorch-YOLOv3
 you should make custom parking slot dataset to train,
 annotations format : [classid     x_center     y_center   bbox_w   bbox_h   vacant]
 
class:
 
right-angled_head  : 0
 
acute-angled_head  :  1
 
obtuse-angled_head : 2
 
marking_point : 3
 
x_center:
 
y_center:
 
bbox_w:
 
bbox_h:
 
whether parking slot is vacant:
 
vacant : 0
 
non-vacant : 1
 
marking point : 2
 
angle: 67, 90 , 127
 
—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by CC-Xiaoyu** [comment]
> VPS-Net is follow this project to train : https://github.com/eriklindernoren/PyTorch-YOLOv3 you should make custom parking slot dataset to train, annotations format : [classid x_center y_center bbox_w bbox_h vacant]
> 
> class:
> 
> right-angled_head : 0
> 
> acute-angled_head : 1
> 
> obtuse-angled_head : 2
> 
> marking_point : 3
> 
> x_center:
> 
> y_center:
> 
> bbox_w:
> 
> bbox_h:
> 
> whether parking slot is vacant:
> 
> vacant : 0
> 
> non-vacant : 1
> 
> marking point : 2
> 
> angle: 67, 90 , 127

Can you share your data set? Thank you!

**Comment by hxx160011** [comment]
Thank you ，I use the PS2.0 dataset which was published by Tongji University



发自我的iPhone


------------------ Original ------------------
From: CC-Xiaoyu ***@***.***&gt;
Date: Tue,Nov 30,2021 2:38 PM
To: weili1457355863/VPS-Net ***@***.***&gt;
Cc: hxx160011 ***@***.***&gt;, Author ***@***.***&gt;
Subject: Re: [weili1457355863/VPS-Net] training (#13)





  
VPS-Net is follow this project to train : https://github.com/eriklindernoren/PyTorch-YOLOv3 you should make custom parking slot dataset to train, annotations format : [classid x_center y_center bbox_w bbox_h vacant]
 
class:
 
right-angled_head : 0
 
acute-angled_head : 1
 
obtuse-angled_head : 2
 
marking_point : 3
 
x_center:
 
y_center:
 
bbox_w:
 
bbox_h:
 
whether parking slot is vacant:
 
vacant : 0
 
non-vacant : 1
 
marking point : 2
 
angle: 67, 90 , 127
  
Can you share your data set? Thank you!
 
—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by msddmzy** [comment]
> > VPS-Net 是按照这个项目来训练的：[https](https://github.com/eriklindernoren/PyTorch-YOLOv3) : [//github.com/eriklindernoren/PyTorch-YOLOv3](https://github.com/eriklindernoren/PyTorch-YOLOv3)你应该制作自定义停车位数据集来训练，注释格式：[classid x_center y_center bbox_w bbox_h vacant]
> > 班级：
> > 直角头：0
> > 锐角头：1
> > 钝角头：2
> > 标记点：3
> > x_中心：
> > y_中心：
> > bbox_w:
> > bbox_h：
> > 车位是否空置：
> > 空缺：0
> > 非空缺：1
> > 标记点：2
> > 角度：67、90、127
> 
> 你能分享你的数据集吗？谢谢！

我也想问一下数据集的问题，现在还不知道我该怎么去使用这种数据集，他标注的只有中心点的位置信息，但没有框的信息。

**Comment by msddmzy** [comment]
> 谢谢，我用的是同济大学发自我的iPhone发布的PS2.0数据集
> […](#)
> ------------------ 原文 ------------------ 来自：CC-Xiaoyu ***@***。 ***> 日期：2021 年 11 月 30 日星期二下午 2:38 致：weili1457355863/VPS-Net ***@***.***> 抄送：hxx160011 ***@***.***>，作者***@***.***> 主题：回复：[weili1457355863/VPS-Net] 训练（#13）VPS-Net 是跟着这个项目训练的：[https](https://github.com/eriklindernoren/PyTorch-YOLOv3) : [//github.com/eriklindernoren/PyTorch-YOLOv3](https://github.com/eriklindernoren/PyTorch-YOLOv3)你应该制作自定义停车位数据集来训练，注释格式：[classid x_center y_center bbox_w bbox_h vacant] class: right-angled_head : 0 acute-angled_head : 1 obtuse-angled_head : 2marking_point : 3 x_center: y_center: bbox_w : bbox_w停车位是空的：空：0 非空：1 标记点：2 角度：67、90、127 你能分享你的数据集吗？谢谢！— 您收到此消息是因为您创作了该线程。直接回复此邮件，在 GitHub 上查看，或取消订阅。使用适用于 iOS 或 Android 的 GitHub Mobile 随时随地分类通知。

我可以问一下您是如何使用这个数据集的吗？是直接转换他的标签信息吗？还是自己标注的呀？

**Comment by msddmzy** [comment]
> VPS-Net is follow this project to train : https://github.com/eriklindernoren/PyTorch-YOLOv3 you should make custom parking slot dataset to train, annotations format : [classid x_center y_center bbox_w bbox_h vacant]
> 
> class:
> 
> right-angled_head : 0
> 
> acute-angled_head : 1
> 
> obtuse-angled_head : 2
> 
> marking_point : 3
> 
> x_center:
> 
> y_center:
> 
> bbox_w:
> 
> bbox_h:
> 
> whether parking slot is vacant:
> 
> vacant : 0
> 
> non-vacant : 1
> 
> marking point : 2
> 
> angle: 67, 90 , 127

可以问一下您是怎么使用这个数据集的呀？是转换ps2.0数据集里面的标签文件，还是直接自己标注的呀？

**Comment by hxx160011** [comment]
转化的标签



发自我的iPhone


------------------ Original ------------------
From: msddmzy ***@***.***&gt;
Date: Mon,Dec 27,2021 5:16 PM
To: weili1457355863/VPS-Net ***@***.***&gt;
Cc: hxx160011 ***@***.***&gt;, Author ***@***.***&gt;
Subject: Re: [weili1457355863/VPS-Net] training (#13)





  
谢谢，我用的是同济大学发自我的iPhone发布的PS2.0数据集
 …
 ------------------ 原文 ------------------ 来自：CC-Xiaoyu @。 &gt; 日期：2021 年 11 月 30 日星期二下午 2:38 致：weili1457355863/VPS-Net @.&gt; 抄送：hxx160011 @.***@***.***&gt; 主题：回复：[weili1457355863/VPS-Net] 训练（#13）VPS-Net 是跟着这个项目训练的：https : //github.com/eriklindernoren/PyTorch-YOLOv3你应该制作自定义停车位数据集来训练，注释格式：[classid x_center y_center bbox_w bbox_h vacant] class: right-angled_head : 0 acute-angled_head : 1 obtuse-angled_head : 2marking_point : 3 x_center: y_center: bbox_w : bbox_w停车位是空的：空：0 非空：1 标记点：2 角度：67、90、127 你能分享你的数据集吗？谢谢！— 您收到此消息是因为您创作了该线程。直接回复此邮件，在 GitHub 上查看，或取消订阅。使用适用于 iOS 或 Android 的 GitHub Mobile 随时随地分类通知。
  
我可以问一下您是如何使用这个数据集的吗？是直接转换他的标签信息吗？还是自己标注的。
 
—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by msddmzy** [comment]
> 转化的标签 发自我的iPhone
> […](#)
> ------------------ Original ------------------ From: msddmzy ***@***.***&gt; Date: Mon,Dec 27,2021 5:16 PM To: weili1457355863/VPS-Net ***@***.***&gt; Cc: hxx160011 ***@***.***&gt;, Author ***@***.***&gt; Subject: Re: [weili1457355863/VPS-Net] training (#13) 谢谢，我用的是同济大学发自我的iPhone发布的PS2.0数据集 … ------------------ 原文 ------------------ 来自：CC-Xiaoyu @。 &gt; 日期：2021 年 11 月 30 日星期二下午 2:38 致：weili1457355863/VPS-Net @.&gt; 抄送：hxx160011 @.***@***.***&gt; 主题：回复：[weili1457355863/VPS-Net] 训练（#13）VPS-Net 是跟着这个项目训练的：https : //github.com/eriklindernoren/PyTorch-YOLOv3你应该制作自定义停车位数据集来训练，注释格式：[classid x_center y_center bbox_w bbox_h vacant] class: right-angled_head : 0 acute-angled_head : 1 obtuse-angled_head : 2marking_point : 3 x_center: y_center: bbox_w : bbox_w停车位是空的：空：0 非空：1 标记点：2 角度：67、90、127 你能分享你的数据集吗？谢谢！— 您收到此消息是因为您创作了该线程。直接回复此邮件，在 GitHub 上查看，或取消订阅。使用适用于 iOS 或 Android 的 GitHub Mobile 随时随地分类通知。 我可以问一下您是如何使用这个数据集的吗？是直接转换他的标签信息吗？还是自己标注的。 — Reply to this email directly, view it on GitHub, or unsubscribe. Triage notifications on the go with GitHub Mobile for iOS or Android. You are receiving this because you authored the thread.Message ID: ***@***.***&gt;

谢谢你这么快就回复我了，请问你是转化成yolo训练的xml的标签吗？我目前也想转换成YOLO可以训练的xml文件，但还没成功。想向您请教一下哈，可有偿哈，我的微信是：18323161010

**Comment by 1234dgfg** [comment]
> > VPS-Net is follow this project to train : https://github.com/eriklindernoren/PyTorch-YOLOv3 you should make custom parking slot dataset to train, annotations format : [classid x_center y_center bbox_w bbox_h vacant]
> > class:
> > right-angled_head : 0
> > acute-angled_head : 1
> > obtuse-angled_head : 2
> > marking_point : 3
> > x_center:
> > y_center:
> > bbox_w:
> > bbox_h:
> > whether parking slot is vacant:
> > vacant : 0
> > non-vacant : 1
> > marking point : 2
> > angle: 67, 90 , 127
> 
> 可以问一下您是怎么使用这个数据集的呀？是转换ps2.0数据集里面的标签文件，还是直接自己标注的呀？
你好，请问您弄明白了吗，我也有相关疑惑，是否可以加个联系方式向您请教？vx：243844142
